docs/src/pages/components/autocomplete/Tags.js

```diff
@@ -58,6 +58,8 @@ export default function Tags() {
         options={top100Films.map(option => option.title)}
         defaultValue={[top100Films[13].title]}
         freeSolo
+        autoHighlight
```

Comment:
It is only for dev and test purpose. I'll remove it once PR will be ready for unit tests.

Reply:
Why changing the demos?